src/main/osd/osd_elements.c

```diff
@@ -514,7 +544,7 @@ static void osdElementAverageCellVoltage(osdElementParms_t *element)
 {
     const int cellV = getBatteryAverageCellVoltage();
     element->buff[0] = osdGetBatterySymbol(cellV);
```

Comment:
osdGetBatterySymbol is common for cell and main battery, will move it to osdFormatVoltage()